curriculum/challenges/english/25-front-end-development/lecture-importance-of-accessibility-and-good-html-structure/672a54a6675c168faa84252d.md

```diff
@@ -32,13 +32,13 @@ Here is an example:
 </button>
 ```
 
-In this case, a screen reader may announce this button as `"Search, button"`, even though the button only contains an icon. The `aria-label` attribute tells screen readers what text to use in place of the icon.
+In this case, a screen reader may announce this button as `Search, button`, even though the button only contains an icon. The `aria-label` attribute tells screen readers what text to use in place of the icon.
 
 If the button contained the text “Search” instead of an icon, then there would be no need for the `aria-label` attribute as the text would provide the label for the button.
```

Comment:
```suggestion
If the button contained the text "Search" instead of an icon, then there would be no need for the `aria-label` attribute as the text would provide the label for the button.
```